Jasper Overall/JS Homework 01/homework.js

```diff
@@ -0,0 +1,60 @@
+console.log('Is this thing on?');
+//CACULATOR PART 1
+
+//Solution
+
+function squareNumber(num) {
+  var squaredNumber = num * num;
+  consle.log('The result of squaring the number ' + num + ' is ' + squaredNumber);
+  return squaredNumber;
+}
+
+squareNumber(3);
+
+function halfof(num); {
+var half = num / 2;
+console.log('half of ' + num + ' is ' + half);
+}
+
+//
+const squareNumber = function (num){
+var squaredNumber = num * num;
+consle.log('The result of squaring the number ' + num + ' is ' + squaredNumber);
+return squaredNumber;
+}
+
+//halfNumber
+
+halfOf(5);
+
+function percentOf(num1, num2) {
+  var percent = (num1/num2)* 100;
+  console.log(num1 + ' is ' + percent + '% of ' +num2);
+  return percent;
+}
+
+//Percentof
+
+percentOf(5, 10);
+
+function areaOfCircle(radius) {
+  car area =Matth.PI * squareNumber(radius);
```

Comment:
`car area`?? Do you mean `const area`? 
Also keep in mind spacing here, should be: 
` const area = Matth.PI * squareNumber(radius);`